packages/wdio-allure-reporter/tests/suite.test.ts

```diff
@@ -622,10 +682,14 @@ for (const protocol of ['webdriver', 'devtools']) {
             reporter.onTestPass()
             reporter.onSuiteEnd(suiteEnd())
             reporter.onRunnerEnd(runnerEnd())
-            const results = getResults(outputDir)
+
+            const { results } = getResults(outputDir)
+            const consoleAttachments = results[0].attachments.filter(
```

Comment:
please assert length before you get value

Reply:
:+1: